std/file.d

```diff
@@ -2809,7 +2809,7 @@ version(StdDdoc)
     struct DirEntry
     {
         /++
-            Constructs a DirEntry for the given file (or directory).
+            Constructs a $(D DirEntry) for the given file (or directory).
```

Comment:
I've noticed that too and submitted a Bugzilla ticket: https://issues.dlang.org/show_bug.cgi?id=17687
I chose to classify it as an enhancement, since I'm not sure whether or not this is intended behavior.